xbmc/video/windows/VideoFileItemListModifier.cpp

```diff
@@ -81,7 +81,8 @@ void CVideoFileItemListModifier::AddQueuingFolder(CFileItemList& items)
     pItem->SetProperty("numepisodes", watched + unwatched); // will be changed later to reflect watchmode setting
     pItem->SetProperty("watchedepisodes", watched);
     pItem->SetProperty("unwatchedepisodes", unwatched);
-    pItem->SetProperty("watchedepisodepercent", watched * 100 / (watched + unwatched));
+    pItem->SetProperty("watchedepisodepercent",
```

Comment:
i‘d calculate `const int totalEpisodes = watched + unwatched` right before the whole „SetProperty“ block.